### ISSUE TITLE
conditional initramfs for tpm setup 

### ISSUE THREAD

**Issue by bsherman** [bsherman/ublue-custom#27]
per @KyleGospo ..
> TPM unlock script works great, I just modified it to use initramfs-etc instead of enabling the full thing

so, I thought we should make it conditional on a pre-built initramfs vs default... 

He suggested removing this chunk with something simpler...
```
## Update initramfs to respect tpm2 unlock
rpm-ostree initramfs |grep tpm2 > /dev/null
if [ 0 -eq $? ]; then
  echo "TPM2 already present in rpm-ostree initramfs config."
  rpm-ostree initramfs
  echo
  echo "Re-running initramfs to pickup changes above."
fi
rpm-ostree initramfs --enable --arg=--force-add --arg=tpm2-tss
```

@m2giles comments that no modifications to initramfs are needed in this script assuming the initramfs was pre-built with tpm2 included.

So the script needs conditionally check this.

**Comment by bsherman** [comment]
After this script works conditionally, add as a new recipe to `ujust` in  https://github.com/ublue-os/config

**Comment by bsherman** [comment]
This was closed by e663db83ef6a5bf9e6ef1f85962a63ea10e48b70
